src/ManageCourses.Domain/Migrations/ManageCoursesDbContextModelSnapshot.cs

```diff
@@ -532,6 +555,15 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasPrincipalKey("OrgId");
                 });
 
+            modelBuilder.Entity("GovUk.Education.ManageCourses.Domain.Models.NctlOrganisation", b =>
+                {
+                    b.HasOne("GovUk.Education.ManageCourses.Domain.Models.McOrganisation", "McOrganisation")
+                        .WithMany("NctlOrganisations")
+                        .HasForeignKey("OrgId")
+                        .HasPrincipalKey("OrgId")
+                        .OnDelete(DeleteBehavior.Cascade);
```

Comment:
I'm not convinced we should be deleting on cascade. As I understand it, an `NctlOrganisation` can exist without a `McOrganisation` and vice versa.

Reply:
:-1: NctlOrganisation can't exist without McOrganisation, as it is metadata on McOrganisation, hence the cascade

Reply:
ok, we'll just need to ensure that we have a McOrganisation for every org with an allocation, whenever we update allocations info.